content/guides/ai-studio/create-agents.md

```diff
@@ -0,0 +1,82 @@
+---
+rank: 2
+related_guides:
+  - authentication/tokens/developer-tokens/
+  - box-ai/ai-tutorials/ask-questions
+  - box-ai/ai-tutorials/default-agent-overrides
+  - box-ai/ai-tutorials/generate-text
+  - box-ai/ai-tutorials/extract-metadata-structured
+  - box-ai/ai-tutorials/extract-metadata
+---
+# Create agents
+
```

Comment:
Add legal info: Box AI Studio is available only for Enterprise Advanced accounts.